From 2a487c1f003c1083dfb56d1ab2cb485c0f904f53 Mon Sep 17 00:00:00 2001
From: Punit Soni <punits@qti.qualcomm.com>
Date: Wed, 6 May 2015 18:05:29 -0700
Subject: [PATCH] mm-camera: remove excessive logging

- disable verbose logs
- use syslog instead of stderr for most logs
---
 mm-camera2/includes/camera_dbg.h                                    | 6 +++---
 .../media-controller/modules/imglib/components/include/img_dbg.h    | 2 +-
 mm-camera2/media-controller/modules/pproc-new/cpp/cpp_hardware.c    | 2 +-
 mm-camera2/media-controller/modules/sensors/module/port_sensor.c    | 2 +-
 mm-camera2/media-controller/modules/sensors/module/sensor_dbg.h     | 6 +++---
 5 files changed, 9 insertions(+), 9 deletions(-)

diff --git a/mm-camera2/includes/camera_dbg.h b/mm-camera2/includes/camera_dbg.h
index 5bcfe59..75656d3 100755
--- a/mm-camera2/includes/camera_dbg.h
+++ b/mm-camera2/includes/camera_dbg.h
@@ -43,9 +43,9 @@
 #else
   #define CDBG_HIGH(fmt, args...) fprintf(stderr, fmt"\n", ##args)
   #define CDBG_ERROR(fmt, args...) fprintf(stderr, fmt"\n", ##args)
-  #define CDBG_LOW(fmt, args...) fprintf(stderr, fmt"\n", ##args)
-  #define CDBG(fmt, args...) fprintf(stderr, fmt"\n", ##args)
-  #define ALOGE(fmt, args...) fprintf(stderr, fmt"\n", ##args)
+  #define CDBG_LOW(fmt, args...) do {} while (0)
+  #define CDBG(fmt, args...) do {} while (0)
+  #define ALOGE(fmt, args...) do {} while (0)
 #endif
 
 #define PRINT_2D_MATRIX(X,Y,M) ({ \
diff --git a/mm-camera2/media-controller/modules/imglib/components/include/img_dbg.h b/mm-camera2/media-controller/modules/imglib/components/include/img_dbg.h
index 2a435a7..7ecfbaa 100755
--- a/mm-camera2/media-controller/modules/imglib/components/include/img_dbg.h
+++ b/mm-camera2/media-controller/modules/imglib/components/include/img_dbg.h
@@ -8,7 +8,7 @@
 #include <stdio.h>
 
 #ifndef IDBG_LOG_LEVEL
-#define IDBG_LOG_LEVEL 2
+#define IDBG_LOG_LEVEL 1
 #endif
 
 #ifndef IDBG_LOG_TAG
diff --git a/mm-camera2/media-controller/modules/pproc-new/cpp/cpp_hardware.c b/mm-camera2/media-controller/modules/pproc-new/cpp/cpp_hardware.c
index dca60a5..d47ba81 100644
--- a/mm-camera2/media-controller/modules/pproc-new/cpp/cpp_hardware.c
+++ b/mm-camera2/media-controller/modules/pproc-new/cpp/cpp_hardware.c
@@ -852,7 +852,7 @@ static int32_t cpp_hardware_process_frame(cpp_hardware_t *cpphw,
   }
 #endif
 
-  CDBG_ERROR("%s:%d] stream_status:%p, iden:0x%x, cur_frame_id:%d\n",
+  CDBG_HIGH("%s:%d] stream_status:%p, iden:0x%x, cur_frame_id:%d\n",
     __func__, __LINE__, stream_status,
     stream_status->identity, hw_params->frame_id);
 
diff --git a/mm-camera2/media-controller/modules/sensors/module/port_sensor.c b/mm-camera2/media-controller/modules/sensors/module/port_sensor.c
index a7adfa3..c3c9b21 100644
--- a/mm-camera2/media-controller/modules/sensors/module/port_sensor.c
+++ b/mm-camera2/media-controller/modules/sensors/module/port_sensor.c
@@ -789,7 +789,7 @@ static boolean port_sensor_handle_aec_update(
       if (stats_get && ((stats_get->aec_get.real_gain[0] !=
           stats_update->aec_update.real_gain) ||
          (stats_get->aec_get.lux_idx != stats_update->aec_update.lux_idx))) {
-        SERR("g %f lux idx %f", stats_update->aec_update.real_gain,
+        SLOW("g %f lux idx %f", stats_update->aec_update.real_gain,
           stats_update->aec_update.lux_idx);
         stats_get->aec_get.real_gain[0] = stats_update->aec_update.real_gain;
         stats_get->aec_get.lux_idx = stats_update->aec_update.lux_idx;
diff --git a/mm-camera2/media-controller/modules/sensors/module/sensor_dbg.h b/mm-camera2/media-controller/modules/sensors/module/sensor_dbg.h
index d36f226..183b564 100755
--- a/mm-camera2/media-controller/modules/sensors/module/sensor_dbg.h
+++ b/mm-camera2/media-controller/modules/sensors/module/sensor_dbg.h
@@ -38,8 +38,8 @@
   #else
     #include <stdio.h>
     #define SERR(fmt, args...) fprintf(stderr, fmt"\n", ##args)
-    #define SLOW(fmt, args...) fprintf(stderr, fmt"\n", ##args)
-    #define SHIGH(fmt, args...) fprintf(stderr, fmt"\n", ##args)
+    #define SLOW(fmt, args...) do{}while(0)
+    #define SHIGH(fmt, args...) do{}while(0)
   #endif
 #elif defined(SLOG_HIGH)
   #ifdef _ANDROID_
@@ -57,7 +57,7 @@
     #include <stdio.h>
     #define SERR(fmt, args...) fprintf(stderr, fmt"\n", ##args)
     #define SLOW(fmt, args...) do{}while(0)
-    #define SHIGH(fmt, args...) fprintf(stderr, fmt"\n", ##args)
+    #define SHIGH(fmt, args...) do{}while(0)
   #endif
 #else
   #ifdef _ANDROID_
-- 
1.9.1

